api/v1alpha3/machine_types.go

```diff
@@ -37,6 +37,16 @@ const (
 
 	// MachineDeploymentLabelName is the label set on machines if they're controlled by MachineDeployment
 	MachineDeploymentLabelName = "cluster.x-k8s.io/deployment-name"
+
+	// PreDrainDeleteHookAnnotionPrefix annotation specifies the prefix we
+	// search each annotation for during the pre-drain.delete lifecycle hook
+	// to pause reconciliation.
```

Comment:
```suggestion
	// to pause deletion.
```
to avoid confusion with the pause reconciliation at cluster/object level

Reply:
We are in effect pausing reconciliation, not just deletion.  Returning empty, nil results in the object not being reconciled further.  Makes more sense to me anyway :)